### PR TITLE
Fixing up some mobile issues that we saw when using the site in mobile

### PR DESCRIPTION
**Pull request by krisfrenette**
This is referencing issue #37 
- Fixed the menu background
- Fixed the news break points
- Fixed the buttons so that they overflow and wrap properly

### CODE REVIEW

File: css/bootstrap/_navbar.scss [hunk 1]
```diff
@@ -64,6 +61,7 @@
   padding-left: 0;
   margin-bottom: 0;
   list-style: none;
+  background-color: white;
```
Comment (tbekolay) [inline css/bootstrap/_navbar.scss]: I'm pretty sure this file is part of bootstrap, so can we do any custom stuff in the (admittedly poorly named) `bootstrap.scss` file? I think that's where all the other custom stuff lives.

**Review by tbekolay — CHANGES_REQUESTED** [review]
One small issue I noted in an inline comment. Also, it would be nicer if applying prettier to these files could be done in a separate commit from the actual changes to make it more obvious where the changes are, but I wouldn't block the PR on that (so just separate it out if it won't take too long).

**Comment by krisfrenette** [comment]
Sorry about the formatting, I should have looked at it. Everything has been fixed.

**Review by tbekolay — APPROVED** [review]
Looks great, thanks! Merging now.

**Pull request merged**